db/schema.rb

```diff
@@ -1318,7 +1318,7 @@
 
   add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
   add_foreign_key "active_storage_variant_records", "active_storage_blobs", column: "blob_id"
-  add_foreign_key "microservice_registrations", "Competitions", column: "competition_id"
+  add_foreign_key "microservice_registrations", "Competitions", column: "competition_id", on_update: :cascade
```

Comment:
Now you were just lazy and didn't actually run it, so the delete cascade doesn't show in `schema.rb` :P

Reply:
:innocent: